src/main/java/net/sf/jabref/gui/StringDialog.java

```diff
@@ -61,6 +61,8 @@
 
 class StringDialog extends JDialog {
 
+    private static final long serialVersionUID = 3649022684378600168L;
```

Comment:
Can you please change your warning settings in eclipse? We never serialize an instance in a File or over the Network and therefore do not need these IDs.